components/compositing/compositor.rs

```diff
@@ -1152,9 +1396,10 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
             MouseWindowEvent::MouseUp(_, p) => p,
         };
 
-        let result = match self.hit_test_at_device_point(point) {
-            Some(result) => result,
-            None => return,
+        let Some(result) = self.hit_test_at_device_point(point) else {
+            // TODO notify embedder?
+            // let msg = ConstellationMsg::UnconsumedEvent(...);
+            return;
```

Comment:
I would remove the commented out code here.

In general, we should be notifying the embedder when a click isn't handled, but also when it hits a WebView, but ultimately isn't consumed as well, I think.

Reply:
done in 4f22eec48373d3596c3c70ca7932fcdff91883ee